great_expectations/rule_based_profiler/data_assistant/onboarding_data_assistant.py

```diff
@@ -615,7 +615,7 @@ def _build_datetime_columns_rule() -> Rule:
 
         column_partition_parameter_builder_for_metrics: ParameterBuilder = DataAssistant.commonly_used_parameter_builders.build_partition_parameter_builder(
             name="column_values.partition",
-            bucketize_data=True,
+            bucketize_data=False,
```

Comment:
What does this do?

Reply:
@NathanFarmer This is an old directive -- it essentially nudges the system to use `histogram` versus `partitioning` metric.  For `DateTimes` we want partitioning, hence the change.  Thanks!

Reply:
Leaving a comment here for future discussion, possibly with @jcampbell

- `histogram`  / `partitioning` are the paradigms that we can use, but it's not so clear 
- `bucketize_data` = True --> ???  I understand is this is `histogram`, but it may be less of a mental leap if we made the naming more consistent.